q2_feature_classifier/_vsearch.py

```diff
@@ -0,0 +1,71 @@
+# ----------------------------------------------------------------------------
+# Copyright (c) 2016-2017, QIIME 2 development team.
+#
+# Distributed under the terms of the Modified BSD License.
+#
+# The full license is in the file LICENSE, distributed with this software.
+# ----------------------------------------------------------------------------
+
+import pandas as pd
+from q2_types.feature_data import (
+    FeatureData, Taxonomy, Sequence, DNAFASTAFormat)
+from .plugin_setup import plugin
+from qiime2.plugin import Int, Str, Float, Choices, Range
+from ._consensus_assignment import (_consensus_assignments,
+                                    _get_default_unassignable_label)
+
+
+def vsearch(query: DNAFASTAFormat, reference_reads: DNAFASTAFormat,
+            reference_taxonomy: pd.Series, maxaccepts: int=10, min_id: int=0.8,
+            strand: str='both', min_consensus: float=0.51,
+            unassignable_label: str=_get_default_unassignable_label(),
+            num_threads: str=1) -> pd.DataFrame:
+
+    seqs_fp = str(query)
+    ref_fp = str(reference_reads)
+    cmd = ['vsearch', '--usearch_global', seqs_fp, '--id', str(min_id),
+           '--strand', strand, '--maxaccepts', str(maxaccepts),
+           '--maxrejects', '0', '--output_no_hits', '--db', ref_fp,
+           '--threads', str(num_threads), '--blast6out']
```

Comment:
``num_threads`` was an attempt at standardizing nomenclature with ``blast``, as described above, now irrelevant and renamed ``threads``.

``id`` appears to be a built-in function in python so I used ``min_id`` instead of being a naughty programmer. I have renamed this ``id_``, which I've seen others use elsewhere. Is that okay?